src/editors/utils/convertMarkdownToXML.js

```diff
@@ -0,0 +1,541 @@
+const convertMarkdownToXml = (markdown) => {
+  const demandHintTags = [];
+
+  // Comprehensive XML conversion function
+  const toXml = (partialMarkdown) => {
+    let xml = partialMarkdown;
+    let i; let makeParagraph; let demandhints;
+    const responseTypes = [
+      'optionresponse', 'multiplechoiceresponse', 'stringresponse', 'numericalresponse', 'choiceresponse',
+    ];
+
+    // fix DOS \r\n line endings to look like \n
+    xml = xml.replace(/\r\n/g, '\n');
+
+    // replace headers
+    xml = xml.replace(/(^.*?$)(?=\n==+$)/gm, '<h3 class="hd hd-2 problem-header">$1</h3>');
+    xml = xml.replace(/\n^==+$/gm, '');
+
+    // extract question and description(optional)
+    // >>question||description<< converts to
+    // <label>question</label> <description>description</description>
+    xml = xml.replace(/>>([^]+?)<</gm, (match, questionText) => {
+      const result = questionText.split('||');
+      const label = `<label>${ result[0] }</label>\n`; // xss-lint: disable=javascript-concat-html
+
+      // don't add empty <description> tag
+      if (result.length === 1 || !result[1]) {
+        return label;
+      }
+      // xss-lint: disable=javascript-concat-html
```

Comment:
I think you can remove all these `xss-lint` comments? We don't have this linter in React/MFE repos and don't need it.